### ISSUE TITLE
NVDA silent uninstall

### ISSUE THREAD

**Issue by jfayre** [nvaccess/nvda#14603]
### Is your feature request related to a problem? Please describe.
NVDA does not offer a way to perform a silent uninstall
### Describe the solution you'd like
NVDA offers silent installs via the command line. The same should be offered for silent uninstall. This is critical for corporate situations where NVDA is managed via SCCM.

**Comment by josephsl** [comment]
Context: stems from a Twitter discussion.

**Comment by jfayre** [comment]
I got it working! The uninstall switch is case sensative. uninstall.exe /S
Closing this issue.

**Comment by XLTechie** [comment]
Single letter commandline switches almost always are, except in the situation of 
Microsoft programs that have been around a long time, like dir and such.

Of course this is a generalization, but it is usually a safe assumption that 
single letter switches are case sensitive.
